test/NuGet.Clients.Tests/NuGet.CommandLine.Test/RestoreNETCoreTest.cs

```diff
@@ -9114,7 +9114,49 @@ public async Task RestoreNetCore_PackagesLockFileWithFloatingVersion_LockedModeI
                 result.Success.Should().BeTrue();
                 Assert.True(File.Exists(projectA.AssetsFileOutputPath));
                 Assert.True(File.Exists(projectA.NuGetLockFileOutputPath));
+            }
+        }
+
+        [Fact]
+        public async Task RestoreNetCore_PackageReferenceWithAliases_IsReflectedInAssetsFileAsync()
+        {
+            using (var pathContext = new SimpleTestPathContext())
+            {
+                // Set up solution, project, and packages
+                var solution = new SimpleTestSolutionContext(pathContext.SolutionRoot);
+                var framework = "net472";
```

Comment:
filename says `NETCore`, method name says `RestoreNetCore`, but TFM is `net472`? 😀

Reply:
I can rename the file name if you want me to :D
These are really "Restore PackageReference" tests.